.github/workflows/build.yml

```diff
@@ -74,73 +76,24 @@ jobs:
           GENERATE_VERSION: ${{ github.event_name != 'push' || github.repository != 'winglang/wing' }}
         run: pnpm changelog
 
-      - name: Build
-        run: pnpm build:ci
+      - name: Build and Package
+        run: pnpm package --filter=!hangar
         env:
           SEGMENT_WRITE_KEY: ${{ secrets.SEGMENT_WRITE_KEY }}
 
-      - name: Package
-        run: pnpm package:ci
-
-      - name: Upload Wing CLI
-        uses: actions/upload-artifact@v3
-        with:
-          name: wing
-          path: apps/wing/*.tgz
-
-      - name: Upload Wing Console Server
-        uses: actions/upload-artifact@v3
-        with:
-          name: wingconsoleserver
-          path: apps/wing-console/console/server/*.tgz
-
-      - name: Upload Wing Console Design System
-        uses: actions/upload-artifact@v3
-        with:
-          name: wingconsoledesignsystem
-          path: apps/wing-console/console/design-system/*.tgz
-
-      - name: Upload Wing Console UI
+      - name: Upload Dist Artifacts
         uses: actions/upload-artifact@v3
         with:
-          name: wingconsoleui
-          path: apps/wing-console/console/ui/*.tgz
-
-      - name: Upload Wing Console App
-        uses: actions/upload-artifact@v3
-        with:
-          name: wingconsoleapp
-          path: apps/wing-console/console/app/*.tgz
-
-      - name: Upload WingSDK
-        uses: actions/upload-artifact@v3
-        with:
-          name: wingsdk
-          path: libs/wingsdk/*.tgz
-
-      - name: Upload WingCompiler
-        uses: actions/upload-artifact@v3
-        with:
-          name: wingcompiler
-          path: libs/wingcompiler/*.tgz
-
-      - name: Upload WingC WASM
-        uses: actions/upload-artifact@v3
-        with:
-          name: wingc
-          path: target/wasm32-wasi/release/wingc.wasm
-
-      - name: Derive appropriate SHAs for base and head for `nx affected` commands
-        id: setSHAs
-        uses: nrwl/nx-set-shas@v3
+          name: dist
+          path: dist/*.tgz
 
       - name: Check git diff of VSCode Extension
         id: git-diff-vscode-wing
         run: |
-          if git diff --quiet ${{ steps.setSHAs.outputs.base }} ${{ github.sha }} -- apps/vscode-wing; then
-            echo "::set-output name=diff::false"
+          if git diff --quiet origin/${{ github.base_ref }} ${{ github.sha }} -- apps/vscode-wing; then
+            echo "diff=false" >> "$GITHUB_OUTPUT"
```

Comment:
This step intentionally used https://github.com/nrwl/nx-set-shas, regardless of the nx connection. That step got the latest successful SHA.
Either add https://github.com/nrwl/nx-set-shas back in or do it in a more turbo-y way.